roles/confluent.common/tasks/ubuntu.yml

```diff
@@ -40,6 +40,7 @@
   when:
     - repository_configuration == 'confluent'
     - install_java|bool
+  throttle: 1
```

Comment:
what does this do exactly? 
https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html#restricting-execution-with-throttle

You'll notice in the redhat.yml retries on some tasks that fail.

Reply:
It was my solution to a contention on the APT lock when running a deployment on a single node for 2 connect workers. Will switch to the retry to be consistent. Should do the same thing.

Reply:
omg that makes sense now! makes me think we need to add throttle more places then. There are many yum/apt installs right?